braket_features/Getting_Devices_and_Checking_Device_Properties.ipynb

```diff
@@ -0,0 +1,990 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Properties of devices\n",
+    "\n",
+    "The properties of a device can be checked with the `device.properties` call. In the following examples, we show some useful properties one can access for each device. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The supported operations of SV1 are ['ccnot', 'cnot', 'cphaseshift', 'cphaseshift00', 'cphaseshift01', 'cphaseshift10', 'cswap', 'cy', 'cz', 'h', 'i', 'iswap', 'pswap', 'phaseshift', 'rx', 'ry', 'rz', 's', 'si', 'swap', 't', 'ti', 'unitary', 'v', 'vi', 'x', 'xx', 'xy', 'y', 'yy', 'z', 'zz']\n",
+      "\n",
+      "The supported result types are [ResultType(name='Sample', observables=['x', 'y', 'z', 'h', 'i', 'hermitian'], minShots=1, maxShots=100000), ResultType(name='Expectation', observables=['x', 'y', 'z', 'h', 'i', 'hermitian'], minShots=0, maxShots=100000), ResultType(name='Variance', observables=['x', 'y', 'z', 'h', 'i', 'hermitian'], minShots=0, maxShots=100000), ResultType(name='Probability', observables=None, minShots=1, maxShots=100000), ResultType(name='Amplitude', observables=None, minShots=0, maxShots=0)]\n",
+      "\n",
+      "The maximum number of qubits supported by this device is 34\n",
+      "The shots range of this device is (0, 100000)\n",
+      "The price of running tasks on this device: price=4.5 unit='hour'\n"
+     ]
+    }
+   ],
+   "source": [
+    "# the managed simulator SV1\n",
+    "device = AwsDevice(\"arn:aws:braket:::device/quantum-simulator/amazon/sv1\")\n",
+    "\n",
+    "support_gates = device.properties.action['braket.ir.jaqcd.program'].supportedOperations\n",
+    "support_result_types = device.properties.action['braket.ir.jaqcd.program'].supportedResultTypes\n",
+    "qubit_count = device.properties.paradigm.qubitCount\n",
+    "shorts_range = device.properties.service.shotsRange\n",
+    "device_cost = device.properties.service.deviceCost\n",
+    "\n",
+    "print(f'The supported operations of {device.name} are {support_gates}\\n')\n",
+    "print(f'The supported result types are {support_result_types}\\n')\n",
+    "print('The maximum number of qubits supported by this device is', qubit_count)\n",
+    "print('The shots range of this device is', shorts_range)\n",
+    "print('The price of running tasks on this device:', device_cost)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "For the IonQ and Rigetti devices, apart from the properties shown above, we can also get information of the availablity windows and device calibration data."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The exeuation windows for Aspen-8:\n",
```

Comment:
"The availability windows for Aspen-8:\n",

(Note for writer: we are asked to avoid non-inclusive or otherwise objectionable terms such as execution. )